misc/python/materialize/git.py

```diff
@@ -182,13 +192,34 @@ def fetch(
     if all_remotes:
         command.append("--all")
 
-    if include_tags:
+    fetch_tags = (
+        include_tags == YesNoOnce.YES
+        or (include_tags == YesNoOnce.ONCE and force)
```

Comment:
So ONCE + force = YES? Do we still need force then?

Reply:
Yes, I would like to. `force` was not introduced by me and is not specific to tags; it is the git parameter of the fetch operation.
With respect to tags, we might want to fetch with force when tags have changed.
I will add a comment in the code.